package-lock.json

```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "@grafana/lezer-logql",
-      "version": "0.1.9",
+      "version": "0.1.10",
```

Comment:
This wasn't changed in the PR that was merged. We should still do it here, otherwise it will be a weird mismatch.